pkg/maps/lbmap/ipv4.go

```diff
@@ -115,7 +120,8 @@ func initSVC(params InitParams) {
 			RevNatMapMaxEntries,
 			0, 0,
 			bpf.ConvertKeyValue,
-		).WithCache().WithPressureMetric()
+		).WithCache().WithPressureMetric().
+			WithEvents(option.Config.GetEventBufferConfig(RevNat4MapName))
 	}
 
 	if params.IPv6 {
```

Comment:
Don't we need to the `.WithEvents(...)` for the IPv6 maps as well?

Reply:
Yes, thanks :pray: